sys/arduino/base.cpp

```diff
@@ -58,3 +59,9 @@ void delay(int msec)
 {
     xtimer_usleep(1000 * msec);
 }
+
+int analogRead(int pin)
+{
+    adc_init(pin);
```

Comment:
AFAIK there needs to be a conversion from the Arduino pin number to the actual RIOT specific `adc_t` ADC line here...